### ISSUE TITLE
Provider tries to talk to localhost

### ISSUE THREAD

**Issue by baurmatt** [syseleven/terraform-provider-metakube#1]
Using the following code:

```terraform
terraform {
  required_providers {
    metakube = {
      source  = "syseleven/metakube"
      version = "0.1.1"
    }
  }
}

provider metakube {}
resource "metakube_project" "example_project" {
  name = "foobar"
}
```

Leads to the following error:

```
$ KUBERMATIC_TOKEN="${TOKEN}" terraform apply

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # metakube_project.example_project will be created
  + resource "metakube_project" "example_project" {
      + creation_timestamp = (known after apply)
      + deletion_timestamp = (known after apply)
      + id                 = (known after apply)
      + name               = "foobar"
      + status             = (known after apply)
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

metakube_project.example_project: Creating...

Error: error when creating a project: {"Op":"Post","URL":"https://localhost/api/v1/projects","Err":{"Op":"dial","Net":"tcp","Source":null,"Addr":{"IP":"::1","Port":443,"Zone":""},"Err":{"Syscall":"connect","Err":61}}}

  on main.tf line 11, in resource "metakube_project" "example_project":
  11: resource "metakube_project" "example_project" {
```

**Comment by furkhat** [comment]
I see, we should improve errors on configuration. I am on it. For now you can either set `METAKUBE_HOST` environment variable or specify `host` within provider block like:
```
provider metakube {
    host = "https://address"
}
```

**Comment by baurmatt** [comment]
I think we should just default this to `https://metakube.syseleven.de` :)

https://github.com/syseleven/terraform-provider-metakube/blob/7685ef6b63d556e031427e005ad2bc0258954a91/metakube/provider.go#L42

**Comment by furkhat** [comment]
Good point, thanks! Fix 46cc79e315ad01230bdfebbb308e9647e69dd569
